app/models/heritage.rb

```diff
@@ -353,4 +354,17 @@ def apply_stack
   def delete_stack
     cf_executor.delete
   end
+
+  def validate_ssm_parameters
+    ssm_paths = environments.secrets.select{ |key|
+      key.value_from.start_with?("/barcelona/#{district.name}")
+    }.map(&:value_from)
+
+    ssm_parameter = SsmParameters.new(self.district, "")
+    response = ssm_parameter.get_parameters(ssm_paths)
```

Comment:
The method `get_parameters` retrieves the parameters for the paths given, so it is a layer of abstraction over the request. This method should not magically know that it made a HTTP Request by assigning the result to a variable named `response`. Instead, `get_parameters` should return the data in a form that is independent of whether or not it is retrieved via HTTP, read from disk, etc...

Please change it so it is possible to mock `get_parameters` with a known return structure or datatype.